Source/Lib/Codec/EbDefinitions.h

```diff
@@ -514,6 +514,21 @@ typedef void * EB_HANDLE;
                                             // Target rate and and max buffer size should be set properly even for fixed QP.
                                             // Disabled by default.
 #define DEADLOCK_DEBUG                   0
+#if DEADLOCK_DEBUG
+#include <limits.h>
+// The recommended steps to debug encoding dead lock (hang) issue:
+// 1. Track the POCs in the entrace and exit of the encoding pipeline,
+//    by forcibly printing the 1st DEADLOCK_DEBUG log in RC, and the
+//    last DEADLOCK_DEBUG log in PK;
```

Comment:
minor point - be consistent with the semicolons (;) - either have them at the end of all steps, or just remove them

Reply:
Thanks!